docs/navigation-advanced.md

```diff
@@ -17,15 +18,26 @@ meta -->
 
 This document shows you how to configure the following Luigi features:
 
-* [View groups](#view-groups)
-* [Dynamically changeable paths](#dynamically-changeable-paths)
-* [Contexts](#contexts)
-* [Profile](#profile) 
-* [Context switcher](#context-switcher) 
-* [Product switcher](#product-switcher) 
-* [App switcher](#app-switcher) 
-* [Tab navigation](#tab-navigation)
-* [Additional options](#additional-options)
+- [Advanced navigation](#advanced-navigation)
```

Comment:
I would remove these changes, since it makes the document longer/more confusing. 
Maybe it was autogenerated?